insert.go

```diff
@@ -0,0 +1,54 @@
+//nolint
+//CREDIT: https://siongui.github.io/2017/01/30/go-insert-line-or-string-to-file/
+package main
+
+import (
+	"bufio"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+func File2lines(filePath string) ([]string, error) {
```

Comment:
not exported; should be lower case. applies to a handful of functions throughout